docs/modules/postgresql.md

```diff
@@ -43,6 +44,36 @@ name.
 | `tags` | `map[string]string` | Tags to be applied to new resources, specified as key/value pairs. | N | Tags (even if none are specified) are automatically supplemented with `heritage: open-service-broker-azure`. |
 | `extensions` | `string[]` | Specifies a list of PostgreSQL extensions to install | N | |
 
+####### Provisioning Parameters: basic
```

Comment:
We didn't have an established pattern for this, so I thought this might be a good way to handle these plan specific things. We do have the common params for the service as well, so i left them up above.

Reply:
Seems ok-- at least for now. Docs are easy enough to change later if we find a better way.